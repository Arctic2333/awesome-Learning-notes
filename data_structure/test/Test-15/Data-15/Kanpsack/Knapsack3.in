7  20.0
AAA  4.0  12.0
BBB  2.0  2.0
CCC  2.0  1.0
DDD  1.0  1.0
EEE  10.0  4.0
FFF  9.5  7.5
GGG  6.0  3.5
