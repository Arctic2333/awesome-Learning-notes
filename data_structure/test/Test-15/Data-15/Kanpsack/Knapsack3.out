EEE: 4Kg
CCC: 1Kg
GGG: 3.5Kg
FFF: 7.5Kg
BBB: 2Kg
DDD: 1Kg
AAA: 1Kg
Total price: 30.83
